app/routes.js

```diff
@@ -24,6 +24,16 @@ module.exports = function(app) {
             .exec(resHandler(res));
     });
 
+    // find unique values for given screen and field
+    app.get('/api/unique/:screen/:field', function(req, res) {
+        Sample
+            .find({'screen': req.params.screen})
+            .distinct(req.params.field, function(err, data) {
```

Comment:
Oh. I see the sort. Does js have optional args? ie could you add a "sort=False" arg to resHandler, then call `resHandler(res, true)` here?